docs/overview.md

```diff
@@ -0,0 +1,131 @@
+# TrustBloc hub-auth
+
+## Overview
+
+`hub-auth` enables [Chained Federation](https://apicrazy.com/2014/04/24/cross-domain-identity-patterns-chained-federation-service-broker/)
+in the role of Service Broker. It provides end-user authentication services for other components in the TrustBloc
+ecosystem as well as minimal storage containers for end-user wallets to store bootstrap data.
+
+![overview](./img/overview.svg)
+
+`hub-auth` only supports [OpenID Connect](https://openid.net/specs/openid-connect-core-1_0.html) upstream providers as
```

Comment:
From [docker compose config for wallet](https://github.com/trustbloc/edge-agent/blob/master/test/bdd/fixtures/wallet-web/docker-compose.yml#L112-L113)  in edge-agent, I see the OIDC is pointing to hydra and hub-auth url to hub-auth bin/docker generated in this project.

It would be nice if we can explain the relation between hydra/oidc and hub-auth.

Reply:
Not sure if we need to.

It would be nicer if we deployed a proxy between RPs and hub-auth/hydra such that the former only use one URL.